runtime/print.cc

```diff
@@ -318,4 +318,27 @@ std::ostream& operator<<(std::ostream& os, const std::tuple<const stmt&, const n
   return os;
 }
 
+
+std::ostream& operator<<(std::ostream& os, const raw_buffer* buf) {
+  os << "{base=" << buf->base << ", elem_size=" << buf->elem_size << ", dims={";
+  for (std::size_t d = 0; d < buf->rank; ++d) {
+    const slinky::dim& dim = buf->dims[d];
+    os << "{min=" << dim.min() << ", max=" << dim.max() << ", extent=" << dim.extent()
```

Comment:
We should probably have `operator<<(std::ostream& os, const dim&)` too and use that here.